src/assets/i18n/en.json

```diff
@@ -7,7 +7,9 @@
     "spanish": "ES"
   },
   "links": {
-    "about": "About Us"
+    "about": "About Us",
+    "who-did-this": "¿WHO DID THIS?",
```

Comment:
This is in english, question mark at the beginning should be deleted.

Reply:
Done